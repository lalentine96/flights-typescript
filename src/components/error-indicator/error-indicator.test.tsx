import React from 'react';
import { render, cleanup } from '@testing-library/react';

import ErrorIndicator from './error-indicator';

afterEach(cleanup);

it('should match the snapshot', () => {
    const { asFragment } = render(<ErrorIndicator />);
    expect(asFragment()).toMatchSnapshot();
});


