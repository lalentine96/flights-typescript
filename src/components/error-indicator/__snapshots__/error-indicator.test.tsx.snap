// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="jumbotron text-center bg-white"
  >
    <h1
      class="error-title"
    >
      Sorry! Something goes wrong :(
    </h1>
    <h2>
      Please try to reload this page later
    </h2>
  </div>
</DocumentFragment>
`;
